Source/Eto/Forms/Controls/Drawable.cs

```diff
@@ -197,7 +197,7 @@ public new interface IHandler : Panel.IHandler
 			bool SupportsCreateGraphics { get; }
 
 			/// <summary>
-			/// Creates a new drawable control without specifying a large canvas or not
+			/// Creates a new drawable control without specifying canvas size
```

Comment:
Another stupid modification to be removed.
But to be honest, maybe there should be some hint as to or not to use D2D on such platforms that support direct drawing (or whatever similar system). Maybe not all drawables are to be painted by D2D (in my PR it will be distinquished by having and not having content).